lib/cogctl/actions/relays/delete.ex

```diff
@@ -14,20 +14,28 @@ defmodule Cogctl.Actions.Relays.Delete do
   end
 
   defp do_delete(endpoint, relay_names) do
-    Enum.reduce(relay_names, 0, fn(relay_name, acc) ->
+    results = Enum.reduce(relay_names, %{success: [], failure: []}, fn(relay_name, acc) ->
```

Comment:
You could do something with `Enum.group_by/2` or `Enum.partition/2` to condense this a bit.